mycroft/configuration/__init__.py

```diff
@@ -3,6 +3,5 @@
 from ovos_config.locale import set_default_lf_lang, setup_locale, \
     set_default_tz, set_default_lang, get_default_tz, get_default_lang, \
     get_config_tz, get_primary_lang_code, load_languages, load_language
-from ovos_config.locations import SYSTEM_CONFIG, USER_CONFIG, DEFAULT_CONFIG, \
-    get_xdg_config_locations
+from ovos_config.locations import SYSTEM_CONFIG, USER_CONFIG, DEFAULT_CONFIG
```

Comment:
Can we first add a deprecation warning in case anyone is using this wrapped import?

i.e.

```python
def get_xdg_config_locations(*args, **kwargs):
    from ovos_config.locations import get_xdg_config_locations as _get_xdg_config_locations
    from ovos_utils.log import LOG
    LOG.warning("This reference is deprecated and will be removed in version 0.1.0")
    return _get_xdg_config_locations(*args, **kwargs)
```